.github/actions/start-opensearch/action.yml

```diff
@@ -0,0 +1,127 @@
+name: 'Start OpenSearch'
+description: 'Downloads latest build of OpenSearch, installs a plugin, executes and script and then starts OpenSearch on localhost:9200'
+
+inputs:
+  opensearch-version:
+    description: 'The version of OpenSearch that should be used, e.g "3.0.0"'
+    required: true
+
+  plugin-name:
+    description: 'The the name of the plugin to use, such as security-dashboards-plugin'
+    required: true
+
+  plugin-start-script:
```

Comment:
I know this is currently being used to run the `install_demo_configuration.sh` script, but I wonder if we can make it possible to specify settings to add to `opensearch.yml` before starting up the node. It would be useful to have a generic way of modifying the configuration before starting up the node to test different scenarios. @RyanL1997 is facing a similar problem while supplying custom configuration to `OpenSearch-Dashboards` before starting up which is needed for the aggregation view functional tests since the default configuration has that feature disabled.

Reply:
So you want to be able to specify a file name in opensearch.yml instead of specifying it in the workflow? That should be doable though I am not sure that the setting will really be used for anything but here.

Reply:
I created an issue so that we can consider adding this functionality after the fact: https://github.com/opensearch-project/security/issues/2252